### PR TITLE
PLANNER-615: Workbench: Remove deprecated score definition type setting from solver configuration

### PR DESCRIPTION
**Pull request by mcimbora**

### CODE REVIEW

File: optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-client/src/main/java/org/optaplanner/workbench/screens/solver/client/editor/ScoreDirectorFactoryFormViewImpl.ui.xml [hunk 1]
```diff
@@ -16,34 +16,16 @@
...
-    }
-
-  </ui:style>
-
   <bootstrap:Container fluid="true">
     <bootstrap:Legend text='{i18n.ScoreDirectorFactory}'/>
 
     <bootstrap:Row>
       <bootstrap:Column size="MD_2">
-        <bootstrap:FormLabel text='{i18n.ScoreDefinitionType}'/>
```
Comment (ge0ffrey) [inline optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-client/src/main/java/org/optaplanner/workbench/screens/solver/client/editor/ScoreDirectorFactoryFormViewImpl.ui.xml:38]: Can we also remove this key from the language property files? Or is it used by the domain editor too?

**Comment by ge0ffrey** [comment]
+1 (previous comment is optional)

File: optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-client/src/main/java/org/optaplanner/workbench/screens/solver/client/editor/ScoreDirectorFactoryFormViewImpl.ui.xml (hunk 1 above)
Comment (mcimbora) [inline optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-client/src/main/java/org/optaplanner/workbench/screens/solver/client/editor/ScoreDirectorFactoryFormViewImpl.ui.xml:38]: +1 Good spot

**Comment by mcimbora** [comment]
@ge0ffrey Thanks, PR updated.

**Comment by ge0ffrey** [comment]
ctrl-shift-f in IntelliJ is your friend when refactoring :)

+1 to merge

**Comment by csadilek** [comment]
jenkins retest this.

**Pull request merged**